mmdet3d/models/backbones/pointnet2_sa_msg.py

```diff
@@ -139,14 +145,15 @@ def forward(self, points):
         sa_features = [features]
         sa_indices = [indices]
 
-        out_sa_xyz = []
-        out_sa_features = []
-        out_sa_indices = []
+        out_sa_xyz = [xyz]
```

Comment:
In PN++ for segmentation, we need all the `sa_xyz` and `sa_features`, so I include the first xyz here.
I have checked the codebase, only 3DSSD uses PN2MSG, and it only uses the **last** `out_sa_xyz` and `out_sa_features`, so adding a first element won't affect its performance.